SimPEG/regularization/base.py

```diff
@@ -491,9 +877,7 @@ def cell_gradient(self):
 
     @property
     def reference_model_in_smooth(self) -> bool:
-        """
-        Use the reference model in the model gradient penalties.
-        """
+        # Inherited from BaseRegularization class
```

Comment:
Would be nice to add a docstring since it's a public property
```suggestion
        Whether to include the reference model in the smoothness term.
```